packages/sdk/src/Redemptions.ts

```diff
@@ -7,27 +9,27 @@ export class Redemptions {
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#redeem-voucher
 	 */
-	public redeem(code: string, body?: $FixMe) {
-		return this.client.post(`/vouchers/${encode(code)}/redemption`, body ?? {})
+	public redeem(code: string, body?: T.RedemptionsRedeemBody) {
+		return this.client.post<T.RedemptionsRedeemResponse>(`/vouchers/${encode(code)}/redemption`, body ?? {})
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#list-redemptions
 	 */
-	public list(params?: $FixMe) {
-		return this.client.get('/redemptions', params)
+	public list(params?: T.RedemptionsListParams) {
+		return this.client.get<T.RedemptionsListResponse>('/redemptions', params)
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#vouchers-redemptions
 	 */
 	public getForVoucher(code: string) {
-		return this.client.get(`/vouchers/${encode(code)}/redemption`)
+		return this.client.get<T.RedemptionsGetForVoucherResponse>(`/vouchers/${encode(code)}/redemption`)
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#rollback-redemption
 	 */
-	public rollback(redemptionId: string, params?: $FixMe) {
-		let qs: $FixMe = {}
-		let payload: $FixMe = {}
+	public rollback(redemptionId: string, params?: T.RedemptionsRollbackParams) {
```

Comment:
You should not call `encode` here for second time since `params` are already encoded once in:
https://github.com/voucherifyio/voucherify-js-sdk/blob/main/packages/sdk/src/RequestController.ts#L48